pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -373,36 +388,21 @@ func (c *MaxPDVolumeCountChecker) filterVolumes(volumes []v1.Volume, namespace s
 				return fmt.Errorf("PersistentVolumeClaim had no name")
 			}
 
-			// Until we know real ID of the volume use namespace/pvcName as substitute
```

Comment:
Is the removal of these parts related to the enforcement of volume limits? If not, please don't make the changes in this PR.

Reply:
removal of these bits from existing predicates was because this code path no longer applies. Users can not delete in-use PVCs and users can not delete PV bound to PVCs. I agree that, perhaps this can go in separate PR but removing them here helps me in using same unit tests for both old and new predicates. 

I can move this to separate PR (or do nothing) - in which case I will have to carefully duplicate unit tests, because I do not want old behavior (of safeguarding against deleted pvcs/pv) in new predicate.

Reply:
reverted these bits. We will revisit this later.